### ISSUE TITLE
Module "pdlib" is already loaded in Unknown on line 0

### ISSUE THREAD

**Issue by Hainer1** [matiasdelellis/facerecognition#675]
### Expected behaviour
execute of: 
'php -m | grep pdlib' 
without PHP warning

### Actual behaviour
php -m | grep pdlib
PHP Warning:  Module "pdlib" is already loaded in Unknown on line 0

### Steps to reproduce
1. starting with: [(https://github.com/matiasdelellis/facerecognition/wiki/Installation)]
2. [https://github.com/matiasdelellis/facerecognition/wiki/PDlib-Installation]
3. following the "Easy Way" Ubuntu Jammy installation:
'echo "deb https://repo.delellis.com.ar jammy jammy" > /etc/apt/sources.list.d/20-pdlib.list
wget -qO - https://repo.delellis.com.ar/repo.gpg.key | sudo apt-key add -
apt update
apt install php8.1-pdlib'
4.PDlib configuration I have a mismatch due to that I have PHP 8.1. I expect that I have to add the following in the
nano /etc/php/8.1/cli/php.ini

[pdlib]
extension="pdlib.so"

After adding and reboot the 'php -m | grep pdlib' doesnt work correctly - see above

### Server configuration

* **Operating system:**
Proxmox with VM and Ubuntu 22.04

* **Pdlib version:**
as installed above. How can I check this?

* **PHP version:**
8.1

* **Web server:**
Apache 2
* **Database:**
mariaDB
* **Nextcloud version:**
NC26.0.2

**Comment by matiasdelellis** [comment]
@Hainer1 

> PHP Warning: Module "pdlib" is already loaded in Unknown on line 0

This is just a warning. The extension is already installing the INI, and you are adding a new configuration to enable it..

Beyond the warning, which is ugly, can you continue the configuration process?

**Comment by Hainer1** [comment]
When installing via the Nextcloud application, I get the following message:
"The library pdlib is not available. The library bz2 is not available."

Despite the error message, something is installed in the nextcloud/apps/facerecognition folder. 
I would not have expected that.  

I have tried:
sudo -u www-data php --define apc.enable_cli=1  /var/www/nextcloud/occ app:install facerecognition
get the message:
PHP Warning:  Module "pdlib" is already loaded in Unknown on line 0
facerecognition already installed

I tried to activate the app through the Nextcloud Appstore, but the activation button was greyed out:
sudo -u www-data php --define apc.enable_cli=1  /var/www/nextcloud/occ app:enable facerecognition
get the message:
PHP Warning:  Module "pdlib" is already loaded in Unknown on line 0
App "Face Recognition" cannot be installed because the following dependencies ar                                                                                               e not fulfilled: The library bz2 is not available.

I executed: 
apt install php-bz2
sudo -u www-data php --define apc.enable_cli=1  /var/www/nextcloud/occ app:enable facerecognition

I get the message: Facerecognition 0.9.20 enabled.
The Face Recognition tap is available in the Admin Preferences.

If this is the way to go, and the warning can be ignored, then perhaps this will help other users and the ticket can be closed. I will now follow the procedure https://github.com/matiasdelellis/facerecognition/wiki/Installation#setup , and open another ticket if there are further problems.
